README.md

```diff
@@ -58,6 +58,7 @@ Functional examples are included in the
 | lookerstudio\_report\_url | The URL to create a new Looker Studio report displays a sample dashboard for data analysis |
 | neos\_tutorial\_url | The URL to launch the in-console tutorial for the Analytics Lakehouse solution |
 | workflow\_return\_project\_setup | Output of the project setup workflow |
+| workflow\_return\_stop\_cluster | Output of stopping the PHS cluster |
```

Comment:
I don't think we need to bubble this up to the user - it doesn't necessarily provide anything meaningful.

Reply:
It was created automatically and caused by [this](https://github.com/GoogleCloudPlatform/terraform-google-analytics-lakehouse/pull/75/files#diff-de6c47c2496bd028a84d55ab12d8a4f90174ebfb6544b8b5c7b07a7ee4f27ec7R22). Without this output, the lint shows an error since the data `call_stop_cluster` created but never used.